src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -88,8 +85,6 @@ internal sealed class ActiveSolutionBoundTracker : IActiveSolutionBoundTracker,
 
             CurrentConfiguration = configurationProvider.GetConfiguration();
```

Comment:
I really tried to get rid of this by adding another private variable currentConfiguration and if its null call GetConfiguration. 

However I ran into a variety of issues with the tests and considering how complicated the tests are I dont think it makes sense for me to change it. The main problem came with the RaiseAnalyzersChangedIfBindingChanged() method and the tests expecting it to be setup in a certain way. The whole test class needs a rewrite I think